cleanlab/datalab/internal/adapter/constants.py

```diff
@@ -1,9 +1,18 @@
-DEFAULT_CLEANVISION_ISSUES = [
-    "dark",
-    "light",
-    "low_information",
-    "odd_aspect_ratio",
-    "odd_size",
-    "grayscale",
-    "blurry",
-]  # list of CleanVision issue types considered by default in Datalab may differ from CleanVision package default
+DEFAULT_CLEANVISION_ISSUES = {
+    "dark": {"threshold": 0.32},
+    "light": {"threshold": 0.05},
+    "low_information": {
+        "threshold": 0.15,
+        "normalizing_factor": 0.1,
+    },
+    "odd_aspect_ratio": {"threshold": 0.35},
+    "odd_size": {"threshold": 10.0},
+    "grayscale": {},
+    "blurry": {
+        "threshold": 0.29,
+        "normalizing_factor": 0.01,
+        "color_threshold": 0.18,
+    },
+}  # list of CleanVision issue types considered by default in Datalab may differ from CleanVision package default
```

Comment:
why do you have to include all of thresholds here? Are they purposefully all different than the default threshold values in CleanVision?

If some of these are the same value as default values in CleanVision, is there any way to omit them and have them auto-set to same default value in CleanVision?  Otherwise Datalab will fail to benefit from future improvements to CleanVision, which is highly suboptimal

Reply:
Makes a lot of sense. Made changes to overwrite threshold only for low information